app/src/main/java/ru/alexzdns/fundamentals/homework/ui/adapters/ActorsAdapter.kt

```diff
@@ -14,15 +17,19 @@ class ActorsAdapter(
     var actors: List<Actor>
```

Comment:
Старайся объявлять иммутабельные переменные
```suggestion
    private val actors: List<Actor>
```